pkg/payload/payload.go

```diff
@@ -213,31 +215,34 @@ func LoadUpdate(dir, releaseImage, excludeIdentifier string, includeTechPreview
 	return payload, nil
 }
 
-func shouldExclude(excludeIdentifier string, includeTechPreview bool, profile string, manifest *manifest.Manifest) bool {
+func include(excludeIdentifier string, includeTechPreview bool, profile string, manifest *manifest.Manifest) error {
 	annotations := manifest.Obj.GetAnnotations()
 	if annotations == nil {
-		return true
+		return errors.New("no annotations")
 	}
 
 	excludeAnnotation := fmt.Sprintf("exclude.release.openshift.io/%s", excludeIdentifier)
-	if annotations[excludeAnnotation] == "true" {
-		return true
+	if v := annotations[excludeAnnotation]; v == "true" {
+		return fmt.Errorf("%s=%s", excludeAnnotation, v)
 	}
 
-	featureGateAnnotationValue, featureGateAnnotationExists := annotations["release.openshift.io/feature-gate"]
+	featureGateAnnotation := "release.openshift.io/feature-gate"
```

Comment:
Move it to `const`?

Reply:
This is our one occurrence of this string outside of comments and tests.  Did you want the tests to consume the constant, once we have it?  They are hard-coding `TechPreviewNoUpgrade` today too:

```console
$ git grep release.openshift.io/feature-gate | cat
...
pkg/payload/payload_test.go:                            "release.openshift.io/feature-gate":                           "TechPreviewNoUpgrade",
...
```

and there's already an openshift/api constant for that, so I expected the test-suite to prefer string literals to catch thing like "we typo'ed the constant value".  And if we expect external consumers, I'd rather create the constant in openshift/api, or openshift/library-go instead of asking external consumers to import the CVO's pkg/payload.  Thoughts?